src/OneSignalMessage.php

```diff
@@ -3,32 +3,17 @@
 namespace NotificationChannels\OneSignal;
 
 use Illuminate\Support\Arr;
+use NotificationChannels\OneSignal\Traits\OneSignalHelpers;
 
 class OneSignalMessage
 {
-    /** @var string */
-    protected $body;
-
-    /** @var string */
-    protected $subject;
-
-    /** @var string */
-    protected $url;
-
-    /** @var string */
-    protected $icon;
+    use OneSignalHelpers;
 
     /** @var array */
     protected $data = [];
```

Comment:
can ditch this var now as we're going straight onto the parameter in the `setData` method